packages/vanilla/src/index.js

```diff
@@ -26,10 +26,16 @@ function forEach (list, fn) {
   for (let i = 0; i < list.length; i++) fn(list[i])
 }
 
-function microlink (selector, opts) {
+function microlink (selector, opts, reactTarget) {
```

Comment:
Can you think of a better naming for this parameter @Kikobeats?

Reply:
maybe `rootNode`?